pkg/apis/core/helper/helpers.go

```diff
@@ -38,6 +38,17 @@ func IsHugePageResourceName(name core.ResourceName) bool {
 	return strings.HasPrefix(string(name), core.ResourceHugePagesPrefix)
 }
 
+// IsHugePageResourceValueValid returns true if the resource value of storage is
+// integer multiple of page size.
+func IsHugePageResourceValueValid(name core.ResourceName, quantity resource.Quantity) bool {
```

Comment:
name this so it is clear it is scoped to checking divisibility, e.g. `IsHugePageResourceValueDivisible`